OpenRA.Game/Map/MapGenerationArgs.cs

```diff
@@ -0,0 +1,43 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Primitives;
+
+namespace OpenRA
+{
+	public class MapGenerationArgs
+	{
+		[FieldLoader.Require]
+		public string Uid = null;
+
+		[FieldLoader.Require]
+		public string Generator = null;
+
+		[FieldLoader.Require]
+		public string Tileset = null;
```

Comment:
Reading the code below I had to come back to this because I had misunderstood - this appears to be `TilesetId`?

Reply:
Tileset/TileSet is the standard term used everywhere for terraininfo ids.